versioned_docs/version-0.5.0/enterprise/upgrading.mdx

```diff
@@ -98,12 +98,17 @@ Contact sales@weave.works for a valid entitlements secret. Then apply it to the
 kubectl apply -f entitlements.yaml
 ```
 
+In addition, you will add your git repository token (in this example, GitHub) as a secret. This is used to update the configurations and manifests in the management repository.
```

Comment:
s/will add/will need to add/ ?

Actually is this still required given that we can now do client side auth with GH?

Reply:
As we're still seeing authorization loops, I would recommend this for now. The user experience isn't good with several auths required.